release-notifier/README.md

```diff
@@ -0,0 +1,40 @@
+# Release notifier
+
+Welcome to Release Notifier tool.
+
+Release notifier tool was designed to keep community informed about new releases and updates.
+Currently it supports PostgreSQL only with potential to grow and support other applications.
+
+## Major Release Upgrade Process Description
+
+**Objective:** To streamline the process of initiating and implementing a major release upgrade, ensuring community consensus, documentation alignment, and effective communication.
+
+1. **Initiation of Discussion:**
+   - A community member recognizes the need for a major release upgrade, after receiving notification of a new release and initiates a discussion on the Tractus-X mailing list.
```

Comment:
I would add a bit more context to "major release upgrade" is it about a widely used dependency in Eclipse Tractus-X?

Reply:
Added "A community member recognizes the need for a major release upgrade of a helm chart dependency" does that look better?